src/pages/settings/Wallet/VerifyAccountPage.tsx

```diff
@@ -49,16 +49,16 @@ function VerifyAccountPage({route}: VerifyAccountPageProps) {
             return;
         }
 
-        Navigation.navigate(navigateBackTo);
-    }, [isUserValidated, navigateBackTo]);
+        Navigation.navigate(navigateForwardTo);
+    }, [isUserValidated, navigateForwardTo]);
 
     useEffect(() => {
         if (isValidateCodeActionModalVisible) {
             return;
         }
 
         if (!isUserValidated && navigateBackTo) {
-            Navigation.navigate(ROUTES.SETTINGS_WALLET);
+            Navigation.goBack(navigateBackTo);
         } else if (!navigateBackTo) {
             Navigation.goBack();
```

Comment:
I added a `forwardTo` param because currently, the `backTo` param is being used to navigate after successfully validated (line 52). But in our case, we want `backTo` to be the fallback route when going back.

Currently, when going back, we always go back to `ROUTES.SETTING_WALLET` (line 61). I fixed it by going back based on the `backTo` params. So, I also updated all `ROUTES.SETTINGS_WALLET_VERIFY_ACCOUNT.getRoute` usage to pass `ROUTES.SETTING_WALLET` as the `backTo` (except SettlementButton which we want to use the current active route as the `backTo`).

Reply:
This addition looks good for me, what do you think @bondydaa . It is not what we approved in proposal, so seeking your opinion on it